<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoukaiTabForm</class>
 <widget class="QWidget" name="YoukaiTabForm">
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>GENERAL</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>TYPE_OF_YOKAI</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="youkaiCB"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>NICKNAME</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="nicknameEdit">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QFormLayout" name="formLayout_6">
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>NUM1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="num1SB">
                <property name="maximum">
                 <number>65535</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>NUM2</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="num2SB">
                <property name="maximum">
                 <number>65535</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="levelSB">
                <property name="minimum">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>LEVEL</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>LOAF</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="loafSB">
                <property name="maximum">
                 <number>15</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_5">
              <item row="4" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>EXP_AT_THIS_LEVEL</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="expSB">
                <property name="maximum">
                 <number>999999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>AI</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="AICB"/>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="IDEdit">
                <property name="inputMask">
                 <string notr="true">HHHHHHHH</string>
                </property>
                <property name="text">
                 <string notr="true">0</string>
                </property>
                <property name="maxLength">
                 <number>8</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>ID</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>IV</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="IV_HPSB">
            <property name="maximum">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>STR</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="IV_StrSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>SPR</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="IV_SprSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>DEF</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="IV_DefSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>SPD</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="IV_SpdSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>EV</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>STR</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>SPR</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>DEF</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>SPD</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="CB_HPSB">
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="CB_StrSB">
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="CB_SprSB">
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="CB_DefSB">
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="CB_SpdSB">
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>SPORTS_CLUB</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>STR</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>SPR</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>DEF</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>SPD</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="SC_HPSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="SC_StrSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="SC_SprSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="SC_DefSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="SC_SpdSB">
            <property name="minimum">
             <number>-10</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>83</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="youkaiNumApplyB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>UPDATE_INDEX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateIDButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>FIX_ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fixIVButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>FIX_IV</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
